customTests/azure_cuda_bandwidth.nhc

```diff
@@ -10,139 +10,153 @@
 #Catch error codes that may be thrown by the executable passed as the first
 #input, and if an error code is tripped throw the second input as a message
 catch_error() {
-	declare -g output
-	output=$($1)
-	err_code=$?
-	if [ $err_code -ne 0 ]; then
-		die 1 "\t $2 $err_code" >&2
-		return 1
-	fi
-	return 0
+    declare -g output
+    output=$($1)
```

Comment:
are there any changes in this file besides indenting? just making sure I'm not missing anything.

Reply:
Alright, this makes sense. Yeah I can see that change and it seems fine.